.github/workflows/run_win_system_tests.yml

```diff
@@ -25,4 +25,4 @@ jobs:
     - name: Run System Tests
       run: .\SystemTestsRunner.exe
       working-directory: ./build/Release
-      timeout-minutes: 27
+      timeout-minutes: 50
```

Comment:
Honestly, I'm wondering if we should just drop the timeout. I think the default is[ 360 minutes per job](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes) (e.g, this entire file and its set of tests) and we're just setting a limit for a specific step. Since these are public GitHub runners on a public GitHub repository, I don't think we're actually being charged anything by letting it go longer (see [here](https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration)).

@reckenro or @astarche - Have we ever had a test run issue that the timeout properly saved us time on?

Reply:
Hmm. Actually this runs on a self-hosted runner. Do we have some kind of restriction around the time we can use on those?

Reply:
In #952 @reckenro mentioned 27mins was arbitrary, so I arbitrarily bumped it up to 50

Reply:
The tests in the CI finished in 29mins, so we were just over.. Anyway, I say we just go with 50 for now so it can handle +10mins easily, with an additional 10min buffer at that point

Reply:
Yeah, if I recall the 27 mins was with the idea that we could get two runs under an hour I think because we were charged (or at least we think we were charged) per hour for those self-hosted runners. I don't know if that was ever verified and we need enough time for the tests to run so I think this bump to 50 minutes is reasonable.

Reply:
And to Charlie's point, I don't recall a test run issue where the timeout kicked in and saved us time. But personally, I still like having some timeout around in case it pops up.